client/app/src/pages/admin/questionnaires/fields/fields.component.ts

```diff
@@ -340,4 +340,37 @@ export class FieldsComponent implements OnInit {
   listenToAddFieldFormTemplate() {
     this.showAddQuestionFromTemplate = false;
   }
+
+  importOption(options: any): void {
+    const fileInput = document.createElement('input');
+    fileInput.type = 'file';
+    fileInput.accept = '.json';
```

Comment:
This implementation Seems to be related to requirement RFR-AMM.FRM.FLD.VALUES.UPLD. Where could we find the design technical specification to understand the file format required to test and validate the implementation?